site/content/courses/res-2-006-girls-who-build-cameras-summer-2016/sections/instructor-insights.md

```diff
@@ -49,7 +49,7 @@ More raspberry pi kits per girl, team building activities, hands-on presentation
 Participant Information
 -----------------------
 
-![40 girls participated this workshop when it was facilitated in Summer 2016.](https://open-learning-course-data.s3.amazonaws.com/res-2-006-girls-who-build-cameras-summer-2016/b5e3ad16b0570101aac505192112d129_40.png)
+![40 girls participated this workshop when it was facilitated in Summer 2016.](https://open-learning-course-data-ci.s3.amazonaws.com/res-2-006-girls-who-build-cameras-summer-2016/b5e3ad16b0570101aac505192112d129_40.png)
```

Comment:
it seems like the URLs for a bunch of images other than the thumbnails are changed - is that intentional?

Reply:
It looks like it's pointing at the CI bucket now, rather than prod.  I think this is fine for now, as long as the images exist.